roles/certificates/defaults/main.yml

```diff
@@ -5,7 +5,8 @@
 #       -----BEGIN CERTIFICATE-----
 #       [...]
 #       -----END CERTIFICATE-----
+
 certificates_ca: []
-certificates_ca_path: /usr/local/share/ca-certificates
 certificates_ca_package_name: ca-certificates
-certificates_ca_update_command: /usr/sbin/update-ca-certificates
+certificates_ca_path: ""
+certificates_ca_update_command: ""
```

Comment:
Can be removed if they are defined in ``/vars``

Reply:
They are, absolutely right.